dev/io.openliberty.data/src/io/openliberty/data/repository/Is.java

```diff
@@ -26,58 +29,103 @@ public static enum Op {
         Equal,
         GreaterThan,
         GreaterThanEqual,
-        IgnoreCase(Equal),
+        IgnoreCase(Equal, IGNORE_CASE),
         In,
         LessThan,
         LessThanEqual,
         Like,
-        LikeIgnoreCase(Like),
+        LikeIgnoreCase(Like, IGNORE_CASE),
         Prefixed,
-        PrefixedIgnoreCase(Prefixed),
+        PrefixedIgnoreCase(Prefixed, IGNORE_CASE),
         Substringed,
-        SubstringedIgnoreCase(Substringed),
+        SubstringedIgnoreCase(Substringed, IGNORE_CASE),
         Suffixed,
-        SuffixedIgnoreCase(Suffixed),
-        Not(Equal),
-        NotIgnoreCase(Equal),
-        NotIn(In),
-        NotLike(Like),
-        NotLikeIgnoreCase(Like),
-        NotPrefixed(Prefixed),
-        NotPrefixedIgnoreCase(Prefixed),
-        NotSubstringed,
-        NotSubstringedIgnoreCase(Substringed),
-        NotSuffixed(Suffixed),
-        NotSuffixedIgnoreCase(Suffixed);
+        SuffixedIgnoreCase(Suffixed, IGNORE_CASE),
+        Not(NOT, Equal),
+        NotIgnoreCase(NOT, Equal, IGNORE_CASE),
+        NotIn(NOT, In),
+        NotLike(NOT, Like),
+        NotLikeIgnoreCase(NOT, Like, IGNORE_CASE),
+        NotPrefixed(NOT, Prefixed),
+        NotPrefixedIgnoreCase(NOT, Prefixed, IGNORE_CASE),
+        NotSubstringed(NOT, Substringed),
+        NotSubstringedIgnoreCase(NOT, Substringed, IGNORE_CASE),
+        NotSuffixed(NOT, Suffixed),
+        NotSuffixedIgnoreCase(NOT, Suffixed, IGNORE_CASE);
+
+        // The remaining part of this class is here to help Jakarta Data providers
+        // interpret the enumerated constants. It can be removed if it doesn't seem
+        // helpful.
 
         private final Op base;
+
         private final boolean ignoreCase;
-        private final boolean negative;
+
+        private final boolean isNegative;
```

Comment:
```suggestion
        private boolean isNegative;
```

Reply:
Let's leave this final and update the constructors to ensure it is always initialized.